ui/src/components/common/damlTypes.ts

```diff
@@ -59,6 +62,7 @@ export function makeContractInfo<T extends object, K = unknown, I extends string
         key: event.key,
         templateId: event.templateId,
         contractId: event.contractId,
+        signatories: event.signatories,
         contractData: event.payload
```

Comment:
Eventually `ContractInfo` is gonna just be an exact copy of `CreateEvent` 🙈 

It may make sense someday to see if we can remove ContractInfo and use the CreateEvent directly, simplify a lot of code.

Reply:
True... at least the name `ContractInfo` is more intuitive to work with than `CreateEvent` 🤷‍♀️